src/libs/actions/Policy/Policy.ts

```diff
@@ -199,6 +199,12 @@ Onyx.connect({
     callback: (val) => (allRecentlyUsedCurrencies = val ?? []),
 });
 
+let nvpActivePolicyID: OnyxEntry<string>;
```

Comment:
```suggestion
let activePolicyID: OnyxEntry<string>;
```

Reply:
Can you please clean it up? getPrimaryPolicy is not used and the getInvoicePrimaryWorkspace is passing the activePolicy from the component so we can I think just remove that param and use the active policy from the Policy.ts

Thanks!